boards/arm/nrf5340dk_nrf5340/Kconfig.defconfig

```diff
@@ -22,6 +22,21 @@ config BOARD
 config BUILD_WITH_TFM
 	default y if BOARD_NRF5340DK_NRF5340_CPUAPPNS || BOARD_NRF5340PDK_NRF5340_CPUAPPNS
 
+if BUILD_WITH_TFM
+
+# By default, force building TF-M without bootloader support,
+# when running tests. This will allow twister to automate the
+# flashing and execution of Zephyr tests for the Non-Secure
+# board version (applicable for nRF5340 Application core).
```

Comment:
isn't this something that can be handled using an `extra_configs` here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/psa_level_1/sample.yaml
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/tfm_ipc/sample.yaml

Reply:
as example: https://github.com/zephyrproject-rtos/zephyr/blob/a890790592d85d7804f5059618d94e1acaeebe5a/samples/net/sockets/echo_async/sample.yaml#L16-L18

Reply:
@tejlmand I want this to be for _all_ tests in the tree, not only the integration samples. I want to run the full tests suite for nRF5340 NS